### DISCUSSION TITLE
Sample Data Sources

### DISCUSSION THREAD

**Discussion by kennethshsu** [casact/chainladder-python#290] [Q&A]
Where is the source of the sample data that comes pre-loaded by the package? Is it downloaded from somewhere?

I am seeing some numbers in `xyz` not reconciling exactly to the examples in the Fridland text.

```python
xyz = cl.load_sample("xyz")

# Case Reserves
# vs Page 281 https://www.casact.org/sites/default/files/database/studynotes_friedland_estimating.pdf
xyz["Incurred"] - xyz["Paid"]

# Incremental Paid
# vs Page 281 https://www.casact.org/sites/default/files/database/studynotes_friedland_estimating.pdf
xyz["Paid"]
```

**Comment by jbogaardt** [comment]
https://github.com/casact/chainladder-python/tree/master/chainladder/utils/data

**Reply by kennethshsu** [reply]
I mean how or where did you download the original data set? Was any transformation performed before putting the csvs here?

**Reply by jbogaardt** [reply]
I believe I grabbed the data from the R chainladder package.

**Reply by kennethshsu** [reply]
Hmm, I don't think I see `xyz` [here](https://github.com/mages/ChainLadder/tree/master/data), any other guesses?

It's not a big deal, just a few numbers are off, so I was wondering if we did any clean up such as transforming or reformatting before we put them in the util folder.

**Reply by jbogaardt** [reply]
Hmm then idk. Can you correct it?
